openedx/core/djangoapps/theming/helpers_dirs.py

```diff
@@ -0,0 +1,199 @@
+import os
+from path import Path
+from django.conf import settings
```

Comment:
Given that we're importing this module from settings, this import concerns me.  I guess it might be ok as long as we don't call any of the functions that actually ask for a setting to be resolved from a settings module, but that seems risky.  Could we change these to accept the specific settings they need as arguments instead?

Reply:
Seems do-able and less confusing. I'll try to move the methods that need `settings` back over to `helpers.py`.